entities/entity_utilities.coffee

```diff
@@ -0,0 +1,11 @@
+class EntityUtilities
+
+  @findLocation: (parsedToken) ->
+    parsedToken = JSON.parse(JSON.stringify parsedToken)
+    for k, v of parsedToken
+      if typeof(v) is 'object'
```

Comment:
I'm being really pedantic here, but typeof is an [operator, not a function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof), so it should be `typeof v`